content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -58,22 +58,26 @@ kubectl apply -f https://kubernetes.io/examples/controllers/frontend.yaml
 ```
 
 You can then get the current ReplicaSets deployed:
+
```

Comment:
@tengqm 👋 , Just letting you know that there may be overlap here with PR #19088 .

Reply:
Okay, there will be a rebase then.